src/runtime/virtcontainers/qemu_amd64.go

```diff
@@ -30,13 +31,15 @@ const (
 
 	defaultQemuMachineType = QemuPC
 
-	defaultQemuMachineOptions = "accel=kvm,kernel_irqchip"
+	defaultQemuMachineOptions = "accel=kvm,kernel_irqchip=on"
```

Comment:
@jimcadden patch lgtm.
One question though for my understanding. Any specific reason to explicitly set to `kernel_irqchip` to `on` . Afaik with `accel=kvm` this is on by default.  Did it break in some configuration or you just want to make it explicit ?

@dgibson ?

Reply:
@bpradipt Nothing broke and this is not strictly related to SEV. After updating to qemu6, I started to see warnings that `kernel_irqchip` should be replaced with the (explicit) `kernel_irqchip=on`.

Reply:
@bpradipt thanks for the review

Reply:
> > Any specific reason to explicitly set to kernel_irqchip to on
> 
> @bpradipt Nothing broke and this is not strictly related to SEV. After updating to qemu6, I started to see warnings that `kernel_irqchip` should be replaced with the (explicit) `kernel_irqchip=on`.

Thanks for the clarification.

Reply:
Ideally that cleanup would go into a separate patch, or it causes reviewer confusion, as we've seen.